pkgs/top-level/python-packages.nix

```
@@ -1607,6 +1607,13 @@ in {
 
   vdf = callPackage ../development/python-modules/vdf { };
 
+  vdirsyncer = disabledIf (!isPy3k) (callPackage ../development/python-modules/vdirsyncer {
+    inherit (pkgs.darwin.apple_sdk.frameworks) CoreServices Security;
+    inherit (pkgs) pkg-config openssl rustPlatform;
+  });
+
+  vdirsyncerStable = disabledIf (!isPy3k) (callPackage ../development/python-modules/vdirsyncer/stable.nix { });
```

please move disable logic inside the expression
```suggestion
  vdirsyncer = callPackage ../development/python-modules/vdirsyncer {
    inherit (pkgs.darwin.apple_sdk.frameworks) CoreServices Security;
    inherit (pkgs) pkg-config openssl rustPlatform;
  };

  vdirsyncerStable = callPackage ../development/python-modules/vdirsyncer/stable.nix { };
```

---

:trumpet: